docs/includes/pairlists.md

```diff
@@ -227,6 +228,29 @@ The optional `bearer_token` will be included in the requests Authorization Heade
 !!! Note
     In case of a server error the last received pairlist will be kept if `keep_pairlist_on_failure` is set to true, when set to false a empty pairlist is returned.
 
+#### MarketCapPairList
+
+`MarketCapPairList` employs sorting/filtering of pairs by their marketcap rank based of CoinGecko. It will only recognize coins up to the coin placed at rank 250. The number of pairs in the resulted pairlist will be slightly different depends on the `mode` defined in the config (available mode are `top_rank` and `total_assets`). The marketcap data from 
```

Comment:
this kinda looks ... incomplete :laughing: 
though i have no idea what you wanted to say in that last sentence ...

Reply:
indeed. I might have left it and forgot about it